sample-app/src/main/java/com/sinch/sample/mailgun/messages/SendMime.java

```diff
@@ -0,0 +1,90 @@
+package com.sinch.sample.mailgun.messages;
+
+import com.sinch.sample.BaseApplication;
+import com.sinch.sdk.domains.mailgun.api.v1.EmailsService;
+import com.sinch.sdk.domains.mailgun.models.v1.emails.request.SendMimeEmailRequest;
+import com.sinch.sdk.domains.mailgun.models.v1.emails.response.SendEmailResponse;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.logging.Logger;
+
+public class SendMime extends BaseApplication {
+
+  private static final Logger LOGGER = Logger.getLogger(SendMime.class.getName());
+
+  public SendMime() throws IOException {}
+
+  public static void main(String[] args) {
+    try {
+      new SendMime().run();
+    } catch (Exception e) {
+      LOGGER.severe(e.getMessage());
+      e.printStackTrace();
+    }
+  }
+
+  public void run() {
+
+    EmailsService service = client.mailgun().v1().emails();
+
+    LOGGER.info("Send message with Mailgun to: " + mailgunTo);
+
+    SendMimeEmailRequest parameters = createMime();
+
+    SendEmailResponse response = service.sendMime(mailgunDomain, parameters);
+
+    LOGGER.info("Response: " + response);
+  }
+
+  SendMimeEmailRequest createMime() {
+    String mimeBody =
+        """
+From: %s
+Subject: SendMime Email Example
+Content-Type: multipart/alternative; boundary="boundary-string"
+
+--your-boundary
```

Comment:
Nit: does it make sense to have a custom boundary?

Reply:
Good catch.
It have to be `"boundary-string"`
Fixed